Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3047,9 +3047,9 @@ def gen2018HiMix(fragment,howMuch):
 defaultDataSets['2026D35']='CMSSW_10_4_0_mtd3-103X_upgrade2023_realistic_v2_2023D35noPU_2-v'
 defaultDataSets['2026D41']='CMSSW_10_6_0_patch2-106X_upgrade2023_realistic_v3_2023D41noPU-v'
 
-keys=defaultDataSets.keys()
-for key in keys:
-  defaultDataSets[key+'PU']=defaultDataSets[key]
+puDataSets = {}
```

Comment:
for my own understanding, what in python3 makes this change necessary?

Reply:
python3 complain about change in dict size 
```
RuntimeError: dictionary changed size during iteration
```

Reply:
Ah, so it must be a change in how `.keys()` behaves. Good to know, thanks.